tensorflow/core/kernels/mkl_conv_grad_filter_ops.cc

```diff
@@ -300,19 +300,24 @@ template <typename T>
 class MklConvBwdFilterPrimitiveFactory : public MklPrimitiveFactory<T> {
  public:
   static MklConvBwdFilterPrimitive<T>* Get(
-      const MklConvBwdFilterParams& convBwdFilterDims) {
+      const MklConvBwdFilterParams& convBwdFilterDims, bool not_cache) {
```

Comment:
Please rename all instances of 'not_cache' into 'do_not_cache'.

Reply:
Sure

Reply:
Code change has been done with variable renaming in all occurrences. 
Thanks!